Remotion/Web/Core/UI/Controls/WebButtonType.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Remotion.Web.UI.Controls
+{
+  /// <summary>
+  /// Defines how a <see cref="WebButton"/> is displayed on the page.
+  /// </summary>
+  public enum WebButtonType
```

Comment:
Sorry, I just realized that we want to resue the ButtonType also in DropDownMenu, so we should keep using `ButtonType` as originally specified instead of `WebButtonType`.